src/binder/bind/bind_ddl.cpp

```diff
@@ -38,16 +41,46 @@ static void validateUniquePropertyName(const std::vector<PropertyInfo>& infos) {
     }
 }
 
+static std::unique_ptr<parser::ParsedFunctionExpression> createSerialDefaultExpr(std::string name) {
+    std::string literalRaw = "'" + name + "'";
```

Comment:
You can leave literalRaw and functionRaw empty. We keep these information to throw the original input when an exception happens. Since this expr is created by us and exception is not expected, we don't care about `xxxRaw`